test/testdata/goldens/get/hosted/advisory_test/more than one affected package with pub as ecosystem.txt

```diff
@@ -0,0 +1,11 @@
+# GENERATED BY: test/get/hosted/advisory_test.dart
+
+## Section 0
+$ pub get
+Resolving dependencies...
```

Comment:
Probably has to be regenerated after https://github.com/dart-lang/pub/pull/4175

Reply:
Yup, done.